tests/utils/shippable/shippable.sh

```diff
@@ -70,6 +70,8 @@ else
     export ANSIBLE_COLLECTIONS_PATHS="${PWD}/../../../"
 fi
 
+retry ansible-galaxy -vvv collection install amazon.aws
```

Comment:
@jillr do we want amazon.aws coming from galaxy, or from galaxy.  My feeling is git so we can work on changes that need to happen across repos.